plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1083,8 +1180,8 @@ public void testAlias()
                 "SELECT count(*) FROM orders");
     }
 
-    @Test
-    public void testSelectInformationSchemaForMultiIndexAlias()
+    //@Test TODO flaky test
```

Comment:
why is the test flaky ?

Reply:
This is fixed in #8381. Let's wait till #8381 gets merged.

Reply:
See #8381

Reply:
I'll be removing this once it's fixed upstream, I was actually using this PR as a demo to show Trinewbies how the contribution process looks like and wantedall the tests to pass :). Sorry for the tag/untag also @martint. I'll let you know when this PR is ready ready with docs/updated tests, etc...